.claude/skills/multi-repo-dsv4flash-setup/bench_attn.sh

```
@@ -0,0 +1,30 @@
+PTO_ISA_COMMIT=${SIMPLER_PTO_ISA_COMMIT:-83d01313}
+
+source "$ROOT/.venv/bin/activate"
+eval "$(pypto-setup --export)" 2>/dev/null
+# Pin ptoas AFTER pypto-setup --export, whose PATH otherwise wins with the
+# slow default ptoas-bin that times out compiling csa.
+export PTOAS_ROOT="/usr/local/ptoas/$PTOAS_VER"
+export PATH="/usr/local/ptoas/$PTOAS_VER/bin:$PATH"
+export PTO_ISA_ROOT="$ROOT/build/pto-isa" SIMPLER_PTO_ISA_COMMIT="$PTO_ISA_COMMIT"
+export PYPTO_BENCH=1
+
+cd "$ROOT/build/pypto-lib"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)